packages/big-design/src/components/Toggle/styled.tsx

```diff
@@ -0,0 +1,44 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import { hideVisually } from 'polished';
+import styled from 'styled-components';
+
+import { withTransition } from '../../mixins/transitions';
+
+import { ToggleLabelProps } from './Toggle';
+
+export const HiddenCheckbox = styled.input`
+  ${hideVisually()}
+`;
+
+export const StyledToggleLabel = styled.label<ToggleLabelProps>`
+  ${withTransition(['background, border-color'])}
+  background: ${({ checked, theme }) => (checked ? theme.colors.primary30 : theme.colors.secondary30)};
+  border-color: ${({ checked, theme }) => (checked ? theme.colors.primary30 : theme.colors.secondary30)};
+  border-radius: ${({ theme }) => theme.helpers.remCalc(8)};
+  cursor: pointer;
+  height: ${({ theme }) => theme.helpers.remCalc(14)};
+  position: relative;
+  width: ${({ theme }) => theme.helpers.remCalc(38)};
+
+  &::before {
```

Comment:
Didn't know where to put this, but what about stylings for the different states (focused, hover, etc...)?
<img width="1044" alt="Screen Shot 2020-08-03 at 12 05 20" src="https://user-images.githubusercontent.com/10539418/89208469-097eef80-d582-11ea-8a18-a66cc0e44fed.png">

Reply:
Have updated styles as per the designs in figma file.